packages/react-button/src/components/Button/ButtonDisabled.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore - add link to this comment please  (see https://github.com/microsoft/fluentui/pull/18695)
+import { Button } from '@fluentui/react-button';
+import { CalendarMonth24Regular } from '@fluentui/react-icons';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
```

Comment:
Works as expected in codesandbox but the _show code does not show anything outside the exported component.
What is the plan to solve this? Would it make sense to use the babel plugin generated source for _show code_?

Reply:
> show code does not show anything outside the exported component.

I fixed this now by setting parameter `docs.source.code` to `__STORY__`. 
`__STORY__` is a variable which contains the source code. It’s added by Storybook to each `.stories.tsx` file.

> What is the plan to solve this [generally]?

To get it to work, we’d need to create an addon which would hide the original Show code button and add a new one which would show full source. This will happen as a part of https://github.com/microsoft/fluentui/issues/19310.

> Would it make sense to use the babel plugin generated source for show code?

To clarify, the babel plugin in this PR doesn't actually generate code, just makes the `__STORY__` variable accessible from addons.
 
Yes, I think implementation of https://github.com/microsoft/fluentui/issues/19310 will use the babel plugin to access story code.